src/tools/ceph_dedup/ceph_dedup_daemon.cc

```diff
@@ -187,7 +187,7 @@ class SampleDedupWorkerThread : public Thread
       }
     }
 
-    bool contains(string& fp) {
+    bool contains(std::string& fp) {
```

Comment:
```suggestion
    bool contains(const std::string& fp) {
```

Reply:
Sure, if you have other fixups in mind it's fine with me.  My suggestion was based on seeing the conversion from an iterator-based loop to a range-based one which falls outside of strictly prepending `std::`.

Reply:
created #63289